src/modelinstance.cpp

```diff
@@ -632,6 +639,17 @@ void ModelInstance::unloadModel(bool isPermanent) {
     }
 }
 
+const Status ModelInstance::validateNumberOfInputs(const tensorflow::serving::PredictRequest* request, const size_t expectedNumberOfInputs) {
+    if (request->inputs_size() < 0 || expectedNumberOfInputs != static_cast<size_t>(request->inputs_size())) {
+        std::stringstream ss;
+        ss << "Expected: " << expectedNumberOfInputs << "; Actual: " << request->inputs_size();
+        const std::string details = ss.str();
```

Comment:
Unnecessary line -> use ss.str() in spdlog_debug

Reply:
It's used in return statement to include error message in status so we can return it to the client

Reply:
You are right - I missed that it's used twice.